meta-dory/recipes-android/brcm-patchram-plus/brcm-patchram-plus/patchram.service

```diff
@@ -5,7 +5,7 @@ Description=Load firmware into BCM20715A1 bluetooth chip
 Type=simple
 ExecStartPre=/bin/sleep 12
 ExecStartPre=/usr/sbin/rfkill unblock bluetooth
-ExecStart=/usr/bin/brcm_patchram_plus --baudrate 3000000 --enable_lpm --enable_hci --no2bytes --patchram /vendor/firmware/BCM20715A1.hcd /dev/ttyHS99
+ExecStart=/usr/bin/brcm_patchram_plus --baudrate 3000000 --enable_lpm --enable_hci --no2bytes --patchram /vendor/firmware/BCM20702A.hcd /dev/ttyHS99
```

Comment:
Should the Description line (line 2) also be changed to refer to BCM20702A rather than BCM20715A1?

Reply:
Good catch! I've changed it to say `Broadcom` to be a bit more generic as the kernel sources mention another Broadcom chip: `BCM4343W` (https://android.googlesource.com/kernel/msm/+/refs/heads/android-msm-dory-3.10-marshmallow-mr1-wear-release/drivers/bluetooth/nitrous.c#302)